### DISCUSSION TITLE
Failed to visit the websites using IPv6 address

### DISCUSSION THREAD

**Discussion by Sving1024** [dani-garcia/vaultwarden#3655] [Q&A] [answered]
I have already set env "ROCKET_ADDRESS=::0" and "WEBSOCKET_ADDRESS=::0
but i still cant visit the websites.

when i tried to use curl
$ curl http://[::1]:8080
curl: (56) Recv failure: Connection reset by peer.

My command:
$ sudo docker run  -d --restart=always --name vaultwarden -p 8080:80 -p 3012:3012 -e SIGNUPS_ALLOWED=true -e WEB_VAULT_ENABLED=true -e WEBSOCKET_ENABLED=true -e DOMAIN=https://mydomain.cn -e ROCKET_ADDRESS=::0 -e WEBSOCKET_ADDRESS=::0 -v /data/bitwarden:/data vaultwarden/server:latest

**Comment by kpfleming** [comment, marked as answer]
You enabled IPv6 connections *inside* the container, but didn't do anything to tell Docker to forward IPv6 connections into the container. I suspect it does not do that by default, but you'd need to check the Docker documentation to be sure.

**Comment by Sving1024** [comment]
Thanks. I really have a mistake in daemon.json .
After I rectify it,it works very well.
